src/Components/Contact/ContactForm.js

```diff
@@ -74,19 +73,16 @@ const ContactForm = async () => {
     >
       {({ errors, handleBlur, touched }) => (
         <Form className="form-contact">
-          
           <Field
             className="input-field"
             name="name"
             placeholder="Nombre"
             type="text"
             onBlur={handleBlur}
           />
-          {
-          (touched.name && errors.name)
-          ? (<div className="alert-danger">{errors.name}</div>)
-          : null
-          }
+          {touched.name && errors.name ? (
+            <div className="alert-danger">{errors.name}</div>
+          ) : null}
 
```

Comment:
Aca podría ser 

```suggestion
{touched.name && errors.name && <div className="alert-danger">{errors.name}</div>}
```
o bien,

```suggestion
{(touched.name && errors.name) && <div className="alert-danger">{errors.name}</div>} 
```
al igual que en los otros campos.
(Esos enters que aparecen en este comentario, no deberían estar)